test/static_dsl.jl

```diff
@@ -232,7 +238,7 @@ end
 # at_choice_example_1
 ir = Gen.get_ir(typeof(at_choice_example_1))
 i = get_node_by_name(ir, :i)
-ret = get_node_by_name(ir, :ret)
+ret = get_node_by_addr(ir, :x)
```

Comment:
@ztangent Does the call to get_node_by_name raise an error here and in other similar lines in this file?

Reply:
Yeah, because with the new changes, the node named `:ret` is now a `JuliaNode` which basically acts as a wrapper. To get the node corresponding to the `@trace` call (which has a gensym-ed name), you need to look it up by the trace address instead (or you can look it up by the gensym-ed name, but we can't know what that is in advance).